components/Templates/Templates.php

```diff
@@ -395,6 +395,23 @@ public function add_meta_boxes() {
 					'restrict_capability' => true,
 				),
 			),
+			array(
+				'name'       => 'show_restrict_message',
+				'label'      => __( 'Show no access message?', 'pods' ),
+				'default'    => 1,
+				'type'       => 'boolean',
+				'dependency' => true,
+			),
+			array(
+				'name'                  => 'restrict_message',
+				'label'                 => __( 'No access message', 'pods' ),
+				'type'                  => 'wysiwyg',
+				'default'               => __( 'You do not have access to view this content.', 'pods' ),
+				'wysiwyg_editor_height' => 200,
+				'depends-on'            => array(
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 814b59c3b7984cfcac741f040cd3ec63</sub>